### DISCUSSION TITLE
Tags extended support

### DISCUSSION THREAD

**Discussion by ww7** [mikekazakov/nimble-commander#354] [Ideas]
Tagged files/folders that in sync with iCloud is greyed, NC can't detect the color and operate them.

Another thing, context menu shows only default tags, so any interface to manage them would be good. I'm added script that call `Get Info` from Finder for now.

**Comment by mikekazakov** [comment]
Take a look at the Setting->Panel->Tags tab, specifically the "Search for new tags" button.
Apple does not provide APIs to query nor modify the set of tags that MacOS manages, hence Nimble Commander has to go this route.

**Reply by ww7** [reply]
Thanks, for regular files works.

For files that at iCloud unfortunately NC not recognizing tag and showing only as grey.

**Reply by mikekazakov** [reply]
Hm, yes, just checked - this looks like a bug. NC should have been able to extract the correct colour from the file's com.apple.metadata:_kMDItemUserTags.
